benchmarks/400.inference/421.image-captioning/input.py

```diff
@@ -0,0 +1,40 @@
+import glob
+import os
+
+def buckets_count():
+    return (1, 1)
+
+'''
+    Generate test, small, and large workload for image captioning benchmark.
+
+    :param data_dir: Directory where benchmark data is placed
+    :param size: Workload size
+    :param benchmarks_bucket: Storage container for the benchmark
+    :param input_paths: List of input paths
+    :param output_paths: List of output paths
+    :param upload_func: Upload function taking three params (bucket_idx, key, filepath)
+'''
+def generate_input(data_dir, size, benchmarks_bucket, input_paths, output_paths, upload_func):
+    input_files = glob.glob(os.path.join(data_dir, '*.jpg')) + glob.glob(os.path.join(data_dir, '*.png')) + glob.glob(os.path.join(data_dir, '*.jpeg'))
```

Comment:
Minor nitpick: this could be a nice loop over extensions `for ext in ['*.jpg', '*.jpeg', '*.png']` :-)

Reply:
Okay